_includes/codepens/readonly-demo/index.html

```diff
@@ -0,0 +1,26 @@
+<script type="text/javascript">
+  function getEditorStatus (editorId) {
+    return tinymce.get(editorId).mode.get();
+  };
+  function setEditorStatus (editorId, currentStatus) {
+    if (currentStatus === "design") {
+      tinymce.get(editorId).mode.set("readonly");
+    } else {
+      tinymce.get(editorId).mode.set("design");
+    }
+  };
+  function enableDisable (targetEditor, targetElement) {
+    var status = getEditorStatus(targetEditor);
+    var button = document.getElementById(targetElement);
+    setEditorStatus (targetEditor, status);
```

Comment:
One more case here where there's a space in a function call. I also missed this before sorry.

Reply:
fixed by https://github.com/tinymce/tinymce-docs/pull/1601/commits/87dc4ff3bc8862d32151713a7f64b1a116ccfcae